framework/src/actions/CheckOutputAction.C

```diff
@@ -50,10 +50,10 @@ CheckOutputAction::checkVariableOutput(const std::string & task)
   {
     // Loop through the actions for the given task
     const std::vector<Action *> & actions = _awh.getActionsByName(task);
-    for (std::vector<Action *>::const_iterator it = actions.begin(); it != actions.end(); ++it)
+    for (auto act : actions)
```

Comment:
I would also argue that we still want to use `const auto &` here for consistency.

What do you guys think?

The normal thought is that creating a reference takes "longer" for simple intrinsic types... but I like the consistency.  I've tried to look up timing on range-based for loops for simple intrinsic types and I can't find anyone that has compared them.  I guess I'll write my own little test code...